src/cpu/cpu_convolution_list.cpp

```diff
@@ -112,6 +113,7 @@ const std::map<conv_impl_key_t, std::vector<pd_create_f>> impl_list_map {
         CPU_INSTANCE_X64(jit_avx2_convolution_fwd_t)
         CPU_INSTANCE_X64(jit_sse41_convolution_fwd_t)
         CPU_INSTANCE_AARCH64(jit_sve_512_convolution_fwd_t<f32>)
+        CPU_INSTANCE_AARCH64_ACL(acl_indirect_gemm_convolution_fwd_t)
```

Comment:
out of curiosity, why is the `indirect_gemm_comvolution` a separate entry-point in ACL, and not an implementation detail?

Reply:
Hi @mgouicem,

The Compute Library API does provide a [common entry point ](https://arm-software.github.io/ComputeLibrary/v20.11/_n_e_convolution_layer_8cpp_source.xhtml) for the various convolution implementations (direct, GEMM, winograd, indirect GEMM and FFT). For the implementation in oneDNN though, we're calling into the underlying functions for each and keeping the logic for which impl. to use as part of oneDNN - this provides a bit more flexibility which is advantageous given that Compute Library, as it stands, isn't specifically targeting infrastructure-scale CPUs.

Reply:
Good morning, @mgouicem. Right now this is the optimal way, as Nathan has described, the thing may change in future.

Reply:
Thanks for the clarification!